code/espurna/serialsensor.cpp

```diff
@@ -0,0 +1,141 @@
+/*
+
+SERIAL_SENSOR MODULE
+
+Copyright (C) 2017-2021 by Patrizio Bekerle <patrizio at bekerle dot com>
+
+*/
+
+#if SERIAL_SENSOR_SUPPORT
+
+#include "serialsensor.h"
+#include "mqtt.h"
+#include "ws.h"
+
+// -----------------------------------------------------------------------------
+// COMMUNICATIONS
+// -----------------------------------------------------------------------------
+
+/**
+ * Setup function
+ *
+ * I haven't managed to get the terminalRegisterCommand commands in a method
+ * because of the lambda function
```

Comment:
btw there's at least one way to 'proxy' certain dynamic arguments inside of a lambda like this:
https://github.com/xoseperez/espurna/blob/62d83c6194630076fa198ddbf0a899ae7fb79672/code/espurna/sensors/PZEM004TSensor.h#L414-L415
(and true, assuming that this only gets called exactly once)

Reply:
Sneaky, I'll try that. 😁  Thank you.